notes_app/notes/api.py

```diff
@@ -77,6 +78,16 @@ def delete_detail(self, object_list, bundle):
         return bundle.obj.user == bundle.request.user
 
 
+class NoteValidation(Validation):
+    def is_valid(self, bundle, request=None):
+        errors = {}
+        if len(bundle.data[u'title']) == 0:
```

Comment:
check http://python-future.org/unicode_literals.html, strings w/o `u''` is cleaner